sys-devel/clang-common/clang-common-15.0.5.ebuild

```diff
@@ -88,6 +88,20 @@ src_install() {
 		@gentoo-gcc-install.cfg
 	EOF
 
+	# needed until https://reviews.llvm.org/D117181#3266224 is done.
+	if use ppc64; then
+		if use ieee-long-double; then
```

Comment:
```suggestion
	if use ppc64 && use ieee-long-double; then
```

?

Reply:
I've since added ! musl condition and I really really dislike triple or more bash logic trains, so I left third condition on new line like before, but upper condition is double.

Reply:
But… why? It's plain AND, isn't it?